<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PivotMain</class>
 <widget class="QDialog" name="PivotMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pivot Table</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Labels</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Columns</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Rows">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Values">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Columns">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="PageFields">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>141</y>
     <width>92</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Fields</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Labels">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Values</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drag from Labels and Drop in desired field</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>104</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="AddFilter">
      <property name="text">
       <string>Add Filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TotalRows">
      <property name="text">
       <string>Total Rows</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="TotalColumns">
      <property name="text">
       <string>Total Cloumns</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>292</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Options">
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
